shuup/core/pricing/_priceful.py

```diff
@@ -158,6 +159,10 @@ def unit_discount_amount(self):
 
     @property
     def tax_rate(self):
+        return ensure_decimal_places(self.raw_tax_rate)
```

Comment:
I think this is ok if we don't use the `tax_rate` to calculate anything and only for showing the final value. The `raw_tax_rate` should be used to do calculations.